hlx_statics/blocks/header/header.js

```diff
@@ -764,10 +764,10 @@ function globalNavLinkItemDropdown(id, name, links) {
     `;
 }
 
-function globalNavLinkItemDropdownItem(url, name) {
+function globalNavLinkItemDropdownItem(url, name, description) {
   return `
       <li class="spectrum-Menu-item menu-item">
-        <span class="spectrum-Menu-itemLabel"><a href="${url}" class="nav-dropdown-links" daa-ll="${name}" >${name}</a></span>
+        <span class="spectrum-Menu-itemLabel"><a href="${url}" class="nav-dropdown-links" daa-ll="${name}" >${name}<span class="spectrum-Body spectrum-Body--sizeXS">${description}</span></a></span>
```

Comment:
We should only add the description span if description exists:


```suggestion
        <span class="spectrum-Menu-itemLabel"><a href="${url}" class="nav-dropdown-links" daa-ll="${name}" >${name}${description ? `<span class="spectrum-Body spectrum-Body--sizeXS">${description}</span>` : ''}</a></span>
```